modular_skyrat/modules/reagent_forging/code/forge_items.dm

```diff
@@ -294,9 +294,18 @@
 	. = ..()
 	if(length(tool.contents) > 0)
 		user.balloon_alert(user, "tongs are full already!")
-		return
+		return FALSE
 	if(!material_type && !custom_materials)
-		user.balloon_alert(user, "unusable material!")
+		user.balloon_alert(user, "invalid material!")
 		return
 	forceMove(tool)
 	tool.icon_state = "tong_full"
+
+/obj/tong_act(mob/living/user, obj/item/tool)
+	. = ..()
```

Comment:
What's the parent proc of this one? `atom`?

Reply:
yes